algorithm-solution/src/main/java/problemset/TwoSumWithDuplicatesStableSolution.java

```diff
@@ -0,0 +1,24 @@
+/*
+Problem : TwoSum
+Given an array of integers, return indices of the two numbers such that they add up to a specific target.
+You may assume that each input would have exactly one solution, and you may not use the same element twice.
+Example:
+Given nums = [2, 7, 11, 15], target = 9,
+Because nums[0] + nums[1] = 2 + 7 = 9,
+return [0, 1]
+*/
+
+class TwoSumWithDuplicatesStableSolution {
+    public int[] twoSum(int[] nums, int target) {
+        Map<Integer, Integer> pos = new HashMap<>();
+        
+        for(int i = 0; i<nums.length; i++) {
+            if( pos.get(target-nums[i]) != null) {
+              return new int[]{pos.get(target-nums[i]), i};  
+            } else {
```

Comment:
Nice. Just this else is not required.

Reply:
:+1:

Reply:
you mean to remove the else part here?

Reply:
Just asilly thing.
'''java
        for(int i = 0; i<nums.length; i++) {
            if( pos.get(target-nums[i]) != null) {
              return new int[]{pos.get(target-nums[i]), i};  
            } else {
              pos.put(nums[i], i);
            }
        }
'''

Reply:
Now. It's exactly same as the best answer provided in the leetcode now. :+1:

Reply:
Wow! thats great!~ Shall we update the solution and list in the readme?

Reply:
Sure. Prabu!. Let's make this weekend a golden one.